app/models/movie_more.rb

```diff
@@ -28,15 +28,7 @@ def initialize(args)
     @vote_average = args[:vote_average]
   end
 
-  def self.parse_result(result)
-    Movie.find_by(tmdb_id: result[:id]) || tmdb_info(result)
-  end
-
-  def times_seen_by(user)
-    0
-  end
-
-  def self.tmdb_info(result)
+  def self.initialize_from_parsed_data(result)
```

Comment:
> This diff is a little tricky to read, but i'm getting it now. Since we moved the `Movie.find_by` out of this class, we only care about the `tmdb_info` method, which now has a clearer name. 👍

Ah yeah, I should have left a comment on the PR, but yes, that's correct 😅